api-server/storage-test-suite/src/basic.rs

```diff
@@ -483,6 +483,103 @@ where
             assert_eq!(bob_utxos, vec![(outpoint.clone(), output.clone())]);
         }
 
+        // get all utxos
```

Comment:
Please update the comment above for spendable to "spendable" instead of available

Reply:
fixed